zookeeper-server/src/main/java/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -0,0 +1,658 @@
+/*
```

Comment:
### ℹ Getting worse: Primitive Obsession
The ratio of primitive types in function arguments increases from 36,59% to 44,44%, threshold = 30,0%